README.md

```diff
@@ -242,18 +163,20 @@ apt::sources:
     location: 'http://debian.mirror.iweb.ca/debian/'
     release: 'unstable'
     repos: 'main contrib non-free'
-    required_packages: 'debian-keyring debian-archive-keyring'
-    key: '9AA38DCD55BE302B'
-    key_server: 'subkeys.pgp.net'
+    key:
+      id: '9AA38DCD55BE302B'
+      server: 'subkeys.pgp.net'
     pin: '-10'
-    include_src: 'true'
-    include_deb: 'true'
+    include:
+      src: 'true'
+      deb: 'true'
```

Comment:
Careful with the quotes here `'true'` is the string, we need it set to `true` the Boolean.